requests/confirmation_email.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Can you double check this?. I ran this against staging and got:
```
GET,/submitted/confirmation-email/send,Expected a (200) but got a (302) back when getting page: /submitted/confirmation-email/send,7
```

Reply:
As discussed, removed a post in the requests file as my local env for the email limit had been set to 11 not 10.